package.go

```diff
@@ -31,6 +31,30 @@ type PackageFunction struct {
 	Doc      string
 }
 
+// Comment returns the comment explaining the function.
+func (f *PackageFunction) Comment() string {
+	var lines []string
+	if f.Doc != "" {
+		lines = append(lines, "// " + f.Doc)
+	}
+
+	// Adds explanations about C types that are unsafe.Pointer in Go world.
+	// See also https://github.com/go-gl/gl/issues/113.
+	for _, p := range f.Function.Parameters {
+		t := p.Type
+		if t.GoType() == "unsafe.Pointer" && t.Name != "void" && t.Name != "GLvoid" {
+			lines = append(lines, fmt.Sprintf("// Parameter %s has type '%s'.", p.Name, strings.TrimSpace(t.CType())))
```

Comment:
I think it would be better to print the `GoCType` here and below. After all, anyone reading this will be trying to figure out what the corresponding CGO type is, I think?